git-day-1/README.md

```diff
@@ -0,0 +1,263 @@
+
+
+### Do this section in pairs 
+### ⏲️ ( 25 mins )
+
+
+
+Our Git timeline starts off with some commits like this:
+```mermaid
+---
+title: cyf-demo-repo
+---
+gitGraph
+   commit
+   commit
+   commit
+   branch  "week-1-workshop"
+```
+
+We're going to figure out how to 
+
+- make local changes and commit them to our branch "week-1-workshop",
+
+so our history ends up looking something like this:
+
+```mermaid
+---
+title: cyf-demo-repo
+---
+gitGraph
+   commit
+   commit
+   commit
+   branch "week-1-workshop"
+   commit
+   commit
+```
+
+where the most recent commits contain our recent work.
+
+To create a commit, you're going to go through the following steps:
+
+1. Make a change to a file
+2. View the local changes
+2. Stage the changes
+3. Creating the commit
+
+
+### 🗄️ 1. Make a change to a file
+
+1. Open up your local github repo `cyf-demo-repo`.
+2. Go to the Explorer section of VSCode
+3. Locate `file.txt` and edit the file with the answer to the questions.
+4. Remember to save the changes to `file.txt`.
+
+Tip: You can use CMD + S to save changes to a file in VSCode.
+
+
+### 🔬 2. View the local changes
+
+
+We want to view the changes we've just made to our working directory.
+
+1. Go to the **Source Control** tab in VSCode.
+
+![vscode-sidebar-source-control](vscode-sidebar-source-control.png)
+
+2. Go to the Changes section and look at the changes for the file.
+3. Try editing the file again in the Explorer tab and check to see the update is visible in the Source Control panel
+
+
+### 🟢 3. Stage the changes
+
+We need to tell Git which changes we want to be part of our next commit. 
+
+We do this by **staging our changes**.
+
+
+1. Go to the file `README.md` and click on the **+**.
+
+👓 Notice what happens when you carry out step 1.
+
+Explain what you think will happen if you don't stage anything in your working directory.
+
+2. View the **Staged Changes** area in your **Source Control** panel.
+
+
+### 📸 4. Create the commit
+
+Once we've staged changes, then we can **commit** these changes.
+
+📝 Check the number of commits you have locally. They should still be the same as before.
+
+1. Enter a commit message describing briefly what you did in your commit.
+2. Click Commit to create the Git commit.
+
+📝  Now re-check the number of commits you have locally.
+
+### When you're finished...
+
+> ❗ Once you've completed this **commit**, you can swap roles in your pair.
+> Choose another file and then go through the steps in [Creating a commit section](#creating-a-commit)
+
+
+--- 
+
+### Take a break... 🍫
+
+---
+
+## Part 2
+
+### 🫸 Pushing (15 mins)
+
+After committing your work locally, you'll have a local branch that looks like this
+
+```mermaid
+---
+title: cyf-demo-repo
+---
+gitGraph
+   commit
+   commit
+   branch "week-1-workshop"
+   commit
+   commit
+```
+
+
+📋 Double-check you've been committing to your branch `week-1-workshop` branch and not your `main` branch. 
```

Comment:
BTW I just updated the front end so you can use hugo note markup in comments and it will show up on the curriculum